homeassistant/components/icloud/__init__.py

```diff
@@ -81,7 +82,7 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     if entry.unique_id is None:
         hass.config_entries.async_update_entry(entry, unique_id=username)
 
-    icloud_dir = hass.helpers.storage.Store(STORAGE_VERSION, STORAGE_KEY)
+    icloud_dir = Store(hass, STORAGE_VERSION, STORAGE_KEY)
```

Comment:
Side note: Store is probably not appropriate here. The PyiCloudService just wants a directory path.

Reply:
Yeah, it does more of these in different places... I've ignored it, for now, to keep it out of scope.